src/core/annotation.js

```diff
@@ -2147,10 +2194,15 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
       }
     }
 
-    this.checkedAppearance = normalAppearance.get(this.data.buttonValue);
+    this.checkedAppearance =
+      normalAppearance.get(this.data.buttonValue) || null;
     this.uncheckedAppearance = normalAppearance.get("Off") || null;
 
-    this._streams.push(this.checkedAppearance);
+    if (this.checkedAppearance) {
+      this._streams.push(this.checkedAppearance);
+    } else {
+      this._getDefaultCheckedAppearance(params);
```

Comment:
Given that this is a *radio* button, is it actually correct to use same appearance as for the checkbox-case?

Reply:
Sorry, but I'd still like an answer to this my previous question here (regarding if it's "correct" to use this appearance also for a *radio* button, since those often look different from checkboxes).